lib/src/types/geo_types.dart

```diff
@@ -113,10 +113,10 @@ class Box {
   @override
   bool operator ==(Object other) =>
       identical(this, other) ||
-          other is Box &&
-              runtimeType == other.runtimeType &&
-              p1 == other.p1 &&
-              p2 == other.p2;
+      other is Box &&
+          runtimeType == other.runtimeType &&
```

Comment:
IMO it's nicer to just make the class `final` and avoid the `runtimeType` check, but it doesn't make much of a difference. Or just use `typedef LineSegment = (Point, Point);` to begin with.

Reply:
I guess if you make one type `final`, you should make all of them `final` (geo, time, interval, range)? This would be nice but might be a breaking change since `Interval` and `Point` were not `final`. Or just make all the new types final now and wait for other breaking changes when changing `Interval` and `Point`?

Reply:
Yeah, making these final now and the others later is a good choice.